From b7753bde88fbed1e7e3eeb255e405bebab9f487c Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Tue, 6 Mar 2018 15:39:46 +0800
Subject: [PATCH 3449/5242] MLK-17698-3 thermal: imx_sc: add PMIC thermal
 sensors for i.MX8QXP

commit  1d5db2ff97a612c1d63202dc5762bd37e9f0efb9 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add PMIC thermal sensors for i.MX8QXP.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/thermal/imx_sc_thermal.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/thermal/imx_sc_thermal.c b/drivers/thermal/imx_sc_thermal.c
index 2a3aad7..5c6353a 100644
--- a/drivers/thermal/imx_sc_thermal.c
+++ b/drivers/thermal/imx_sc_thermal.c
@@ -49,7 +49,8 @@ enum imx_thermal_trip {
 };
 
 static const sc_rsrc_t imx8qxp_sensor_hw_id[] = {
-	SC_R_SYSTEM, SC_R_DRC_0,
+	SC_R_SYSTEM, SC_R_DRC_0, SC_R_PMIC_0,
+	SC_R_PMIC_1, SC_R_PMIC_2,
 };
 
 const int *sensor_hw_id;
-- 
1.7.9.5

